### ISSUE TITLE
Focus state when pressed doesn't meet contrast on blue buttons

### ISSUE THREAD

**Issue by JuliannaR** [cds-snc/covid-shield-mobile#600]
Similar to this issue: https://github.com/cds-snc/covid-shield-mobile/issues/390

we used: 

Touchable highlight is also limited by opacity. We can't control individual color values of text and background.

Flagging that in react-native 0.63, the Pressablecomponent seems like it can do wonders. @timarney
https://reactnative.dev/docs/pressable

![Image from iOS (68)](https://user-images.githubusercontent.com/35537308/87159333-2c71f880-c28f-11ea-8d46-6c5f71d50d4a.png)
